src/why3prop.ml

```diff
@@ -0,0 +1,61 @@
+(** Translation of lambdapi goals to why3 goals in propositional case *)
+
+open Terms
+open Why3
+
+(* a type to present the list of why3 constants and lambdapi terms *)
+type cnst_table = (term * Term.lsymbol) list
+
+(** [t_goal g] output the translation of a goal [g] into a why3.*)
+let rec t_goal : string -> term -> bool = fun sp g ->
```

Comment:
The comment does not correspond to the type. t_goal takes 2 arguments and returns a boolean.